.github/workflows/build.yml

```diff
@@ -13,58 +13,62 @@ concurrency:
 jobs:
 
   build:
-    strategy:
-      matrix:
-        platform: [ubuntu-latest]
-        go: [1.20.1]
-    name: '${{ matrix.platform }} | ${{ matrix.go }}'
-    runs-on: ${{ matrix.platform }}
+    name: Build
+    runs-on: ubuntu-latest
     steps:
       - name: Check out repository code
         uses: actions/checkout@v3
-      - name: Install go
+
+      - name: Install Go
         uses: actions/setup-go@v3
         with:
-          go-version: ${{ matrix.go }}
+          go-version-file: go.mod
+          check-latest: true
+
       - name: Initialize CodeQL
         uses: github/codeql-action/init@v2
         with:
           languages: go
           queries: security-and-quality
-      - name: Build
+
+      - name: Build executables
         run: |
           GOOS=windows GOARCH=amd64 go build -o build/win-amd64/cbuild.exe ./cmd/cbuild
```

Comment:
We could add arm64 targets as well. I don't expect any issues with the builds.

Reply:
Addressed [Build and archive win & mac arm64 binaries](https://github.com/Open-CMSIS-Pack/cbuild/pull/50/commits/551cefc094f936fc5f1aed661c6b06812f8fec3c)

Reply:
I think you need to change https://github.com/Open-CMSIS-Pack/cbuild/blob/c2fd06c563c4da11dcebb8d7d4e20458c22f2b45/.goreleaser.yml#L12-L16 as well

Reply:
you are right. Done
[Updated go releaser configuration](https://github.com/Open-CMSIS-Pack/cbuild/pull/50/commits/af55469791dbcbf3f68a146b6061b1f6467cf041)